demos/crossroad_camera_demo/main.cpp

```diff
@@ -26,7 +26,9 @@
 #include <samples/slog.hpp>
 #include <samples/ocv_common.hpp>
 #include "crossroad_camera_demo.hpp"
+#ifndef WITHOUT_EXTENSIONS
```

Comment:
This is a double negative. I think it would be better to write it as `#ifdef WITH_EXTENSIONS`.

Reply:
Fixed